app/controllers/v2/content_items_controller.rb

```diff
@@ -12,6 +14,18 @@ def index
       ).call
     end
 
+    def linkables
+      render json: Queries::GetContentCollection.new(
+        content_format: request.query_parameters.fetch(:format),
```

Comment:
The index endpoint uses `content_format` instead of `format`. Would it be good to use that here as well?